TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/SupportTasks/ApiTrnRequests/Resolve/ResolveApiTrnRequestPageModel.cs

```diff
@@ -0,0 +1,181 @@
+using System.Diagnostics;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using TeachingRecordSystem.Core.DataStore.Postgres;
+using TeachingRecordSystem.Core.DataStore.Postgres.Models;
+using TeachingRecordSystem.Core.Models.SupportTaskData;
+using static TeachingRecordSystem.SupportUi.Pages.SupportTasks.ApiTrnRequests.Resolve.ResolveApiTrnRequestState;
+
+namespace TeachingRecordSystem.SupportUi.Pages.SupportTasks.ApiTrnRequests.Resolve;
+
+public abstract class ResolveApiTrnRequestPageModel(TrsDbContext dbContext) : PageModel
+{
+    public JourneyInstance<ResolveApiTrnRequestState>? JourneyInstance { get; set; }
+
+    protected TrsDbContext DbContext => dbContext;
+
+    protected TrnRequestMetadata GetRequestData()
+    {
+        var supportTask = HttpContext.GetCurrentSupportTaskFeature().SupportTask;
+        return supportTask.TrnRequestMetadata!;
+    }
+
+    protected IReadOnlyCollection<PersonMatchedAttribute> GetPersonAttributeDifferences(
```

Comment:
This seems to be returning matches rather than differences?

Reply:
Sure is - I've fixed the name